### PR TITLE
Show loading spinners whilst retriving data

### PR DESCRIPTION
**Pull request by tomneedham**
- [ ] Need to vertically align the spinners

### CODE REVIEW

**Comment by jancborchardt** [comment]
Just use a div to cover the whole area and give it the class `.loading` – that’s how we do it everywhere else as well. ;)

**Comment by tomneedham** [comment]
ok

**Comment by jancborchardt** [comment]
Updates here?

**Comment by tomneedham** [comment]
So as you say the best solution is to add `.loading` to here: https://github.com/owncloud/contacts/blob/master/templates/main.php#L44 

But my Angular knowledge runs out here, @Henni is this even possible as its not in a template?

**Comment by Henni** [comment]
You could probably apply it to the contactlist instead, which has its own controller.
https://github.com/owncloud/contacts/blob/master/templates/main.php#L45

**Comment by tomneedham** [comment]
Ready for review @jancborchardt

**Comment by DeepDiver1975** [comment]
I see a spinner! :tada:

**Comment by DeepDiver1975** [comment]
needs rebase

**Comment by DeepDiver1975** [comment]
:+1:

File: js/services/addressBook_service.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 app.factory('AddressBookService', function(DavClient, DavService, SettingsService, AddressBook, Contact) {
 
 	var addressBooks = [];
+	var loading = true;
```
Comment (irgendwie) [inline js/services/addressBook_service.js]: Do we need this variable here?

**Comment by irgendwie** [comment]
Rebased!
Pls review @Henni @DeepDiver1975

**Comment by Henni** [comment]
LGTM :+1:

**Pull request merged**